widgets/src/components/SummaryCard/__tests__/SummaryCardConfig.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, wait } from '@testing-library/react';
+import React from 'react';
+import 'mocks/i18nMock';
+
+import WIDGET_CONFIGS from 'mocks/app-builder/pages';
+import CONNECTIONS from 'mocks/pda/connections';
+import PROCESSES from 'mocks/pda/processes';
+import SummaryCardConfig from 'components/SummaryCard/SummaryCardConfig';
+
+describe('<SummaryCardConfig />', () => {
+  it('renders snapshot correctly', async () => {
+    fetch
+      .once(JSON.stringify(CONNECTIONS))
+      .once(JSON.stringify(WIDGET_CONFIGS.SUMMARY_CARD))
+      .once(JSON.stringify(PROCESSES));
```

Comment:
Do these show up in the component?

Reply:
`PROCESSES` is not needed, so now I removed it.